qutip/qip/operations/gates.py

```diff
@@ -58,6 +59,114 @@
 #
 
 
+def x_gate(N=None, target=0):
+    """Pauli-X gate or sigmax operator.
+
+    Returns
+    -------
+    result : qobj
```

Comment:
replace `qobj` with `:class: qutip.Qobj`
The later one can be captured by the sphinx documentation automatically which will create an appropriate link to the `Qobj` class.

Reply:
I have a doubt here if you see functions as _check_qubits_oper, _targets_to_list, expand_operator, etc., in the documentation, it says: :class:`qutip.Qobj` with simple quotes. So, I'm not sure if it goes with or without the simple quotes. Let me know if I must change it.

Reply:
Done. :)